mongodbatlas/table_mongodbatlas_database_user.go

```diff
@@ -0,0 +1,163 @@
+package mongodbatlas
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func tableAtlasDatabaseUser(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "mongodbatlas_database_user",
+		Description: "",
+		List: &plugin.ListConfig{
+			Hydrate: listAtlasDatabaseUsers,
+		},
+		Get: &plugin.GetConfig{
+			Hydrate:    getAtlasDatabaseUser,
+			KeyColumns: plugin.AllColumns([]string{"username", "database_name"}),
+		},
+		Columns: []*plugin.Column{
+			{
+				Name: "database_name",
+				Description: `Database against which the database user authenticates. Database users must provide both a username and authentication database to log into MongoDB.
+
+This resource returns:
+
+admin if the database user authenticates with SCRAM-SHA.
+
+If you don't set an authentication mechanism, Atlas defaults to SCRAM-SHA.`,
```

Comment:
```suggestion
				Description: `Database against which the database user authenticates. This resource returns admin if the database user authenticates with SCRAM-SHA. If you don't set an authentication mechanism, Atlas defaults to SCRAM-SHA.`,
```

Are the last 2 sentences relevant/worth including, or would `Database against which the database user authenticates.` be sufficient? Descriptions are generally kept short (1-2 sentences), but we can expand if the information is valuable in understanding the column's values.